src/content/deadlines/00_cfp.mdx

```diff
@@ -1,10 +0,0 @@
----
```

Comment:
did you delete this because it shows on the home page? 😊

Reply:
yeap I'd like to remove that in favor of sponsors. Preview was ok but let me know if I have to update anything else!

Reply:
I've done that in code, so we can keep the deadlines, and only show the last 3 😊